CHANGELOG.md

```diff
@@ -1,4 +1,48 @@
 # Changelog
+## Friday 12th August 2022
+## Docs updates 
+We launched a [Discussion](https://docs.api.audio/discuss) page so you can ask us your questions. And a [FAQs](https://docs.api.audio/discuss?isFAQ=true) page. Check this out!
```

Comment:
''so as you can ask us your questions directly. In addition to that, we have also launched an updated FAQ Page.